api/v1alpha1/driver_types.go

```diff
@@ -119,6 +129,8 @@ type ProvisionerSpec struct {
 
 	// Resource requirements for provisioner's containers
 	Resources ProvisionerResourcesSpec `json:"resources,omitempty"`
+
+	Privileged bool `json:"privileged,omitempty"`
```

Comment:
Please provide a comment to explain this field. and why do we need to give users control over this specific setting?

Reply:
OPenshift defines it as Privileged,
 https://docs.openshift.com/container-platform/4.10/storage/persistent_storage/persistent-storage-hostpath.html#persistent-storage-hostpath-pod_persistent-storage-hostpath
 Do you have any other suggestion?

Reply:
In that case I think we are ok